src/index.native.tsx

```diff
@@ -329,6 +337,14 @@ module.exports = {
 
     return ScreensNativeModules.NativeSearchBar;
   },
+  get OverlayView() {
```

Comment:
I'd perhaps change the name to sth like "FullWindowOverlay" to make it cleaner that it does not just overlay the container where its put or the screen under which it is rendered but instead it displays in front of all the content in the window.

Reply:
`WindowOverlay` could work too.